### ISSUE TITLE
Use MsCeleb-1M and Tripletloss to train，get low accuracy and val_rate

### ISSUE THREAD

**Issue by EmiPark** [davidsandberg/facenet#702]
@davidsandberg 

My training dataset is MsCeleb-1M after cleaned by mtcnn and your text list  provided,in total,data size is about 4.6M.
while I'm training it by train_tripletloss.py with parameter like these:
python src/train_tripletloss.py --logs_base_dir /raid/logs/facenet --models_base_dir /raid/models/facenet  --data_dir MsCeleb_160 --learning_rate -1 --learning_rate_schedule_file data/learning_rate_schedule_classifier_msceleb.txt --lfw_dir raw_160 --lfw_pairs data/pairs.txt  --log_histograms
where learning_rate_schedule_classifier_msceleb.txt set:
0:  0.1
50: 0.01
100: 0.003
200: 0.001
300: 0.0001
Unfortunatelly, my accuracy is top to 0.92 and val_rate is top to 0.46. 
I've tried to change people_per_batch 150 images_per_person 10,but nothing changed.
One thing maybe need to pay attention is that my loss is instability like this:
![image](https://user-images.githubusercontent.com/28434213/38788777-5cbec842-4168-11e8-9375-
1efbb5dd5a9b.png)

I'm very confused by the situation encoutered and very expecting the help.

**Comment by xxllp** [comment]
I think you can use other data
